app/controllers/Application.scala

```diff
@@ -91,4 +91,27 @@ class Application(implicit val wsClient: WSClient, val db: MetricsDB) extends Co
   }
 
   def getNewspaperBookList = APIAuthAction(APIResponse(db.getDistinctNewspaperBooks))
+
+  def getArticlesWithWordCounts() = APIAuthAction { req =>
+    APIResponse {
+      for {
+        articlesWithoutCommissionedLength <- db.getArticlesWithWordCounts(false)(MetricsFilters(req.queryString))
+        articlesWithCommissionedLength <- db.getArticlesWithWordCounts(true)(MetricsFilters(req.queryString))
+      } yield {
+        val resultsOmitted: Boolean = (articlesWithCommissionedLength.length == maxNumberOfArticlesToReturn ||
+          articlesWithoutCommissionedLength.length == maxNumberOfArticlesToReturn)
```

Comment:
This is not strictly correct, by chance we could just have matched the max number of articles we were to return but I don't think this is a big problem and doing it this way keeps the code neater.